### DISCUSSION TITLE
Snapshot beyond 2000px

### DISCUSSION THREAD

**Discussion by coachrye** [percy/cli#437] [Support] [answered]
Hi. 

Anyone know a way to take a snapshot of design beyond the 2000px width limit? 

Client wants to design a website for 27" monitors, which is 2560x1440. 

Tried updating the config.js but the server-side caught it 😅

Thanks in advance.

**Comment by Robdel12** [comment, marked as answer]
Hey @coachrye! 2000px is currently the max width Percy supports -- there's no way to wider than that.

**Reply by coachrye** [reply]
Oh! Got it, @Robdel12 . I guess we'll have to manually test this particular project for now then. Thanks for the quick response.
